pkg/ingester/metrics.go

```diff
@@ -203,6 +207,17 @@ func newIngesterMetrics(r prometheus.Registerer, createMetricsConflictingWithTSD
 		}, []string{"user"})
 	}
 
+	if !createMetricsConflictingWithTSDB {
+		m.numUsersWithBackfillTSDBs = promauto.With(r).NewGauge(prometheus.GaugeOpts{
+			Name: "cortex_ingester_tsdb_users_with_backfill_tsdb",
```

Comment:
I would suggest `cortex_ingester_backfill_tsdb_tenants`, because:
1. We're trying to move from "user" to "tenant" naming (will take ages)
2. We could prefix the backfill metrics with `cortex_ingester_backfill_tsdb`